src/libs/actions/OnyxUpdateManager.ts

```diff
@@ -46,6 +46,7 @@ export default () => {
                 if (isLoadingApp) {
                     console.debug(`[OnyxUpdateManager] Ignoring Onyx updates while OpenApp hans't finished yet.`);
                 }
+                SequentialQueue.unpause();
```

Comment:
Please add a comment to explain why this needs to be here. Also, can you update the comment on line 44 above so that it also explains _why_ updates shouldn't be processed in that situation?